.pre-commit-config.yaml

```diff
@@ -37,7 +37,10 @@ repos:
     rev: v1.7.1
     hooks:
     -   id: mypy
-        additional_dependencies: [types-toml, pydantic]
+        additional_dependencies: [
+            types-toml,
+            tomli, # to read config from pyproject.toml
```

Comment:
**issues:** needs both tomli and pydantic

according to error in precommit, it now reads config from pyproject, but needs pydantic installed

![image](https://github.com/MiraGeoscience/geoapps-utils/assets/24550538/f61647bd-3939-4e74-92f7-da596eb827a8)

```suggestion
            types-toml,
            tomli, # to read config from pyproject.toml
            pydantic,
```